server/middlewares/stats.js

```diff
@@ -41,22 +41,52 @@ const parseTxBodyOutAmount = (txBody) => {
   }
 }
 
-const incrCountersBy = (key, value) => {
-  const [year, month, day] = getSlicedDate()
-  client.incrby(`${key}:total`, value)
-  client.incrby(`${key}:monthly:${year}-${month}`, value)
-  client.incrby(`${key}:daily:${year}-${month}-${day}`, value)
+const initVisitor = (trackingID) => {
+  return ua({
+    tid: trackingID,
+    cid: 555, //google analytics anonymous visitor
+    strictCidFormat: false, //to accept cid integer instead of UUID v4 format
+  })
+}
+
+const trackPageView = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_GA_TRACKING_ID)
+  const args = {
+    dp: argsObj.path,
+    dt: argsObj.hostname,
+    dh: argsObj.title,
+  }
+
+  visitor.pageview(args).send()
+}
+
+const trackEvent = (argsObj) => {
```

Comment:
`argsObj` seems quite cryptic, you can make this more readable by destructuring the object in the function signature: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_function_paramete

i.e. it can be
```
const trackEvent = ({category, action, label, value}) =>
...
```